test/itemlist_formaction.cpp

```diff
@@ -101,8 +100,7 @@ TEST_CASE("process_op(OP_PURGE_DELETED)", "[itemlist_formaction]") {
 	REQUIRE_NOTHROW(itemlist.process_op(OP_PURGE_DELETED));
 }
 
-TEST_CASE("process_op(OP_OPENBROWSER_AND_MARK)", "[itemlist_formaction]") {
-	INFO("Tests item opening in browser by setting the browser option to echo to a temporary file, that is then parsed.");
+TEST_CASE("OP_OPENBROWSER_AND_MARK passes the proper url to the browser and marks read", "[itemlist_formaction]") {
```

Comment:
You don't need "proper" here: within itemview it's pretty obvious that "the url" is item's URL.